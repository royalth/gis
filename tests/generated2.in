20 25
12 10
20 10
20 17
7 4
13 7
1 20
11 5
3 13
20 7
14 16
1 5
14 6
4 9
18 10
14 13
8 15
2 16
19 11
16 8
14 3
18 3
10 6
10 13
12 1
1 17
