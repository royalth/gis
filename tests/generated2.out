5
1 5 2
1 12 3
1 17 4
1 20 1
2 16 2
3 13 2
3 14 3
3 18 4
4 7 3
4 9 2
5 11 1
6 10 1
6 14 2
7 13 1
7 20 2
8 15 2
8 16 3
10 12 2
10 13 3
10 18 5
10 20 4
11 19 2
13 14 4
14 16 1
17 20 3
